### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by ld-xy** [jerpelhan/GeCo#3]
<img width="1039" alt="image" src="https://github.com/user-attachments/assets/c21b264f-7494-43f0-b8a1-8a07ae7208ae" />

**Comment by zupdaz** [comment]
Got the same error, so bboxes have to be provided in advance?

**Comment by jerpelhan** [comment]
Please check whether a Matplotlib window opens first. In this window, you should be able to annotate bounding boxes. If the window does not appear, it is likely, that you are not using the correct backend for Matplotlib.

To check your current backend, run:

```python
import matplotlib
print(matplotlib.get_backend())
```

If needed, try setting a different backend explicitly:

```python
import matplotlib  
matplotlib.use('TkAgg')  # or 'Qt5Agg' depending on your system
```

**Comment by zupdaz** [comment]
Yes, I had the same error within the Google collab notebooks where mathplotlib cannot be openned. 

Can GeCo work as zero-shot also?

**Comment by jerpelhan** [comment]
That is probably the problem. I apologize, but I'm unsure how to run Matplotlib in interactive mode on Google Colab.

Regarding your question, yes, GeCo can run in a zero-shot setting, but only when the task is single-class and it is clear what needs to be counted. Just add ```--zero_shot``` and when calling the model provide an empty tensor for bounding boxes.

**Comment by cripsgreen** [comment]
Hello, I’ve encountered exactly the same issue as you. Have you managed to solve it?
@ld-xy @zupdaz
